Gemfile

```diff
@@ -25,7 +25,9 @@ gem 'resque-timeout', '~> 1.0.0'
 gem 'resque-lock-timeout', '~> 0.4.5'
 gem 'resque-scheduler', '~> 4.3.1'
 # Paperclip is deprecated: https://cm-jira.usa.gov/browse/SRCH-702
-gem 'paperclip', '~> 5.2.0'
+# Using a third-party fork as an interim measure.
+gem 'kt-paperclip', :git => 'https://github.com/kreeti/kt-paperclip.git'
```

Comment:
Is there a reason we need to use the `master` branch instead of a released version: https://rubygems.org/gems/kt-paperclip ? If so, please add a comment that mentions the reason, and update the hash syntax: `git: 'https...`

Reply:
Changed.